.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug Report
+about: Raise an issue about this project
+---
```

Comment:
Can we add labels? i.e.
`labels: tracking issue, needs triage`

Reply:
Will add later on